packages/stream_chat_flutter/pubspec.yaml

```diff
@@ -11,7 +11,8 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
-  stream_chat_flutter_core: ^1.3.2-beta
+  stream_chat_flutter_core:
+    path: ../stream_chat_flutter_core
```

Comment:
I think we can even remove this relative import from here
If you use `melos bootstrap` instead of `flutter pub get` it should work fine